docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -91,15 +89,65 @@ for HTTP and WS.
 
 Authentication tokens expire 5 minutes after being generated. It is necessary to generate a new authentication 
 token if access is required after token expiration.     
+
+## JWT Public Key Authentication
+
+Enable authentication from the command line and supply the public key of the external JWT token. The JWT token must use the `RS256` algorithm.
+
+### 1. Generate a Private and Public Key Pair
+
+The private and accompanying public key file must be in the `.pem` format.
+
+!!! example "Example using OpenSSL"
+    ```bash
+    openssl genrsa -out privateKey.pem 4096
+    openssl rsa -pubout -in privateKey.pem -pubout -out publicKey.pem
+    ```
+
+### 2. Create the JWT Token
+
+Create the JWT token using an external tool. 
+
+!!! important
+    The JWT token must use the `RS256` algorithm 
+
+Each payload for the JWT token must contain:
+
+* [JSON-RPC permissions](#json-rpc-permissions)
+* An expiry period. 
```

Comment:
This is the expiry time claim. Mentioning the claim name I think would be also help remove any ambiguity. So maybe same format as in the rfc something like "exp" (Expiration Time) claim. https://tools.ietf.org/html/rfc7519#section-4.1.4

Reply:
Done.